src/ui/verificationDiagnosticsView.ts

```diff
@@ -0,0 +1,701 @@
+/* eslint-disable max-depth */
```

Comment:
This file has lots of commented code. Is it still work in progress? I'll happily look into it in more detail after it has been cleaned. Even without the commented code file consists of over 500+ lines. There might be opportunities to extract some of the code into separate modules.

Reply:
Yes, I will move this commented out code in another branch.